src/DynamoCore/Library/FunctionDescriptor.cs

```diff
@@ -145,7 +146,12 @@ public FunctionDescriptorParams()
         /// <summary>
         ///     Indicates if the lacing strategy is disabled on the function
         /// </summary>
-        public bool IsLacingDisabled { get; set; } 
+        public bool IsLacingDisabled { get; set; }
+        //TODO - should this somehow contain more info - ExperimentalInfo{IsExperimental, ExperimentalMessage/url}?}
```

Comment:
How would the message be surfaced in your opinion?

Reply:
kind of like how the underlying `system.diagnostics.codeanalysis.experimental` attribute surfaces them. In this case it shows the errors in the errors tab of visual studio as compile time warnings and generates a url given a base url format and the experimental diagnostic id.

we could something similar with the docs browser, though it would be much more useful if you get a link to some real documentation that we update, for example in the forum or primer.

For now I will just leave it as a TODO and file a follow-up to consider it.